### ISSUE TITLE
Lando Pull results in Error

### ISSUE THREAD

**Issue by contour66** [lando/lando#794]
Bug Report
==========

**Tell us about your setup**
MacOS Sierra 10.12.6 

**lando version**

v3.0.0-beta.35 MacOS

**Tell us about your `.lando.yml`**

```name: name
recipe: pantheon
config:
  framework: drupal
  site: some-website
  id: ff2068a8-6662-446b-8dc9-33702fac3868
services:
  appserver:
    xdebug: true
~                                                                                                                                                                                                     
~                                                                                                                                                                                                     
~  
```
**pantheon.yml**              

```api_version: 1
php_version: 7.0
drush_version: 8

protected_web_paths:
  - /CHANGELOG.txt
  - /COPYRIGHT.txt
  - /INSTALL.mysql.txt
  - /INSTALL.pgsql.txt
  - /INSTALL.sqlite.txt
  - /INSTALL.txt
  - /LICENSE.txt
  - /MAINTAINERS.txt
  - /README.txt
  - /UPGRADE.txt
  - /web.config
~                                                                                                                                                                                                     
~                                                                                                                                                                                                     
~  
```
**Tell us about the command you were running**

```lando pull```

**Tell us about the error you got**

![image](https://user-images.githubusercontent.com/22285683/37675989-ee92b9a8-2c33-11e8-938e-6d168022b89b.png)

**Tell us generally about your bug**

Trying to run `lando pull` to get the DB.   Made sure Lando, Docker, Terminus are up to date.
Tried `lando rebuild` as well as `lando destroy` and restarting but still getting the same message.

steps we can take to replicate this bug. 
```bash
1. Lando Start
2. Authorize machine token
3. Lando Pull
```

**Tell us more**

Does this bug prevent you from using lando?

No

**Comment by pirog** [comment]
@contour66 im investigating this but i think we may have deployed new beta.37 docker images into the wrong namespace. I'm going to verify whether that is true and if it is we will revert the images ASAP!

**Comment by contour66** [comment]
Also globally when I run  `terminus -v` it returns `1.7.1` but when I run lando start it is stating I have `v1.7.0` installed 
![image](https://user-images.githubusercontent.com/22285683/37682801-c8a3bcb0-2c47-11e8-9838-c9b084eedda6.png)

**Comment by pirog** [comment]
@contour66 yup, we're behind on the terminus version but will bump to 1.7.1 in the next version

**Comment by pirog** [comment]
@contour66 i'd give a `lando rebuild` a try and see if that does the trick

**Comment by contour66** [comment]
I was able to get past this error I believe it had to do with multiple auth accounts on pantheon that weren't linking up properly. *Note I had to logout of terminus and log back in with the prefferred account. Terminus _will not_ allow for the user to switch accounts by simply using the ` auth login` command.

 However, when I did run `lando pull` it hung forever (approx. 30 min) before I decided to go ahead and cancel and just manually import the db. That's a separate issue so I will go ahead and close this one.
